DataLink.tex

```diff
@@ -1325,25 +1336,34 @@ \section{Changes}
 \subsection{DataLink-1.1}
 
 \begin{itemize}
-\item allow optional columns to contain values when the row (link) has an error\_message
+\item allow optional columns to contain values when the row (link) has an error\_message 
+(see Section \ref{sec:errorMessage})
 \item added optional local\_semantics to identify corresponding rows 
-	for different IDs in the same service
-\item relax content-type usage to allow any valid VOTable MIME type
-\item INFO element with standardID mandatory in \blinks\ response
+	for different IDs in the same service (see Section \ref{sec:localSemantics})
+\item relax content-type usage to allow any valid VOTable MIME type (see Section 
+  \ref{sec:mime})
+\item INFO element with standardID mandatory in \blinks\ response (see Section 
+  \ref{sec:output})
 \item added optional content\_qualifier to describe link target content with terms from
-the product-type vocabulary
+the product-type vocabulary (see Section \ref{sec:contentQualifier})
 \item added optional link\_auth and link\_authorized to signal whether authentication
-is necessary to use the link
-\item clarified use of multiple ID values and possible OVERFLOW
-\item clarified use of utype for self-describing service descriptors
-\item clarified use of semantics
+is necessary to use the link (see Sections \ref{sec:linkAuth} and 
+\ref{sec:linkAuthorized})
+\item clarified use of multiple ID values and possible OVERFLOW (see Section 
+  \ref{sec:resourceId})
+\item clarified use of utype for self-describing service descriptors(see Section 
+  \ref{sec:selfDescribing})
+\item clarified use of semantics (see Section \ref{sect:semantics})
 \item generalize by adding use cases for links to content other than data files
+  (see Sections \ref{sec:useSource} and \ref{sec:useProvenance})
 \item added using LINK to convey when datalink request URL is in a table column
+  (see Section \ref{sec:columnsUrl})
 \item service descriptors can include a contentType param to describe service
-output and should include a name and description
+output and should include a name and description (see Section \ref{sec:descParams})
```

Comment:
Recommendation to use name and description is in sec 4.1 not 4.2 (sec:serviceResources) so that should be referenced here as well.

Reply:
Change made